dmcontent/content_loader.py

```diff
@@ -188,12 +188,24 @@ def get_question_as_section(self, question_slug):
         question = self.get_question_by_slug(question_slug)
         if not question:
             return None
+
+        if question.get('questions'):  # multiquestion
+            return ContentSection(
+                slug=question.slug,
+                name=question.label,
+                editable=self.edit_questions,
+                edit_questions=False,
+                questions=question.questions,
```

Comment:
I think we can merge this with the following constructor into a single one using`questions=question.get('questions', [question])` here.

Editable flags should be the same for both. For G9 sections we need to set `editable=False` and `edit_questions=True`, meaning there should be no way to edit the section itself, only individual questions are editable. That's true for both multiquestions and normal questions and for both `edit_questions=False` since they're the leaf nodes (ie questions inside multiquestions aren't rendered as separate sections).